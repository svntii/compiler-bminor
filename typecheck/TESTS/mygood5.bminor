double:function void (n:integer) = {
    
    for(i = 0; i < n; i = i + 1){
        i = i + i;
        print i + "\n";
    }

}