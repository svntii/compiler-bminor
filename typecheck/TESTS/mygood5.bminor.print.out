//	CSE 40243 B-Minor Parser
//	Enter a valid  B-Minor Code 

//	Scan Successful
//	Parse Successful
double: function void(n: integer)  = 
{
    for (i = 0; i < n; i = i + 1 )
        
        {
            i = i + i;
            print i + "\n";
            
        }
        
}

